.github/update-sync-param-files.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import glob
+
+import yaml
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("output", type=str, help="output path of sync-param-files.yaml")
+    args = parser.parse_args()
+
+    # load current sync-param-files.yaml
+    with open(args.output) as f:
+        sync_param_file_curr = yaml.safe_load(f)
+    src2dst_curr = {}
+    for src_and_dst in sync_param_file_curr[0]["files"]:
```

Comment:
[IMO] Better naming?

```suggestion
    for sync_file_config in sync_param_file_curr[0]["files"]:
```